src/enc.cc

```diff
@@ -22,6 +22,7 @@
 #include <float.h>    // for FLT_MAX
 #include <stdint.h>
 
+#include <cstdlib>
```

Comment:
Probably some more can be done here with iwyu. I'm leaving stdlib.h in this commit to match the change, though I think it's been widely discussed that it would be nearly impossible to remove putting things in the global namespace in the C++ versions of the headers.